index.js

```diff
@@ -282,7 +282,7 @@ function calculateRange(text, opts, ast) {
 }
 
 function formatRange(text, opts, ast) {
-  if (0 < opts.rangeStart || opts.rangeEnd < text.length) {
+  if (opts.rangeStart > 0 || opts.rangeEnd < text.length) {
```

Comment:
I don't like this change, I feel like it's easier to understand with the way it was written before